pkg/telemetry/prometheus/quality.go

```diff
@@ -0,0 +1,47 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/livekit/protocol/livekit"
+)
+
+var (
+	qualityRating prometheus.Histogram
```

Comment:
This is the three bar rating we have. So, won't make much sense to do p50, p90, p95 as there are only three bins (guess we can still do p90, p99. But, probably have to invert the ratings, i. e. if we expect `EXCELLENT` to have the highest number of entries, that is the last bin. Flipping it and making `EXCELLENT` the lowest bin might be more intuitive to think of p90.

But, I am think we should just display each bin as line graph (assuming that is possible) and just see how the three move across time.